testing/libfort/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Pete Dietl <petedietl@gmail.com>
+pkgname=libfort
+pkgver=0.1.3
+pkgrel=0
+pkgdesc="A simple crossplatform library to create formatted text tables."
+url="https://github.com/seleznevae/libfort"
+arch="all"
+license="MIT"
+makedepends="cmake binutils-gold"
```

Comment:
Any particular reason for requiring binutils-gold?

Reply:
I will get rid of this dependency. The upstream author said:

> Hello, I noticed in https://github.com/alpinelinux/aports/pull/8027/files that you have binutils-gold. I then noticed that for all gcc builds I have option -fuse-ld=gold. Is that the reason why you have to add it or something else?
Anyway, as far as I remember I think that I added it because there were some bugs with sanitizers with some gcc versions. Therefore in develop (after the last release) I made some changes - now I add -fuse-gold only for special builds (sanitizers ...).
Not sure this is important. But in case guys from alpine linux say something about binutils-gold maybe it can be avoided for ordinary builds.

So I think he will give me a version 0.1.4.